Update references from claim->statement and ledger-->append-only log (#45)
scitt-community/scitt-web: README.md

In practice, SCITT provides information about artifacts, enabling a mesh of dependencies to understand what each subsystem is consuming.
Detailed information comes in varying formats, from structured to unstructured.

> In SCITT, structured data is represented as a Statement. A Statement is a well-structured statement, made by a verifiable entity that may have supporting evidence.

Reviewer: But it may just be ok, here as the supporting evidence is the signature I suppose!

> In SCITT, structured data is represented as a Statement. A Statement is a well-structured statement, made by a verifiable entity that may have supporting evidence.

Author: The terminology used for "evidence" is another area we should clarify. I kept this PR to focus on claim--> statement.

<img src="./assets/statement-evidence-relationship.png" alt="Identity, Statement, Evidence, Artifact relationship" style="width:300px;"/>

### Continual Updates

Documenting Statements at the time software is built or deployed would sell SCITT short, as software isn't static. Software is continually updated, and more importantly, we continually learn and want to convey new information about artifacts that have already been released. Reputable OSS Projects and Independent Software Vendors (ISVs) don't intentionally produce software they know to be vulnerable. Only after the software is public do we often find out about new vulnerabilities. SCITT is a means to convey a stream of continual updates for each versioned artifact.

## SCITT Persistence

> SCITT is intended to store Transparent Statements for the life of the of the SCITT instance. One of the many questions that surfaces is how big will the SCITT append-only log get? What kind of data will go on the SCITT append-only log that would cause it to grow?

Reviewer: ```suggestion
SCITT is intended to store Signed Statements for the life of the of the SCITT instance. One of the many questions that surfaces is how big will the SCITT append-only log get? What kind of data will go on the SCITT append-only log that would cause it to grow?
```

### eNotary

> SCITT is analogous to a digital, or electronic notary service (eNotary), where minimal information is written to the append-only log, counter-signing the Statement. When users notarize legal documents, the notary append-only log records the verified identity of the parties, referencing the legal document they are notarizing. The notary append-only log doesn't store the legal document, but does have a reference to it. 

Reviewer 2: ```suggestion
SCITT is analogous to a digital or electronic notary service (eNotary), where minimal information is written to the append-only log and an augmented counter-signature over the Statement and corresponding metadata (Receipt) is produced. When users notarize legal documents, the notary append-only log records the verified identity of the parties, referencing the legal document they are notarizing. The notary append-only log doesn't store the legal document, but does have a reference to it. 
```

> In SCITT, the append-only log will contain pointers to the artifact, which Statements are made, with optional pointers to any supporting evidence.

Reviewer 2: Is that found in specification text today?

Author: This is just updating terminology of ledger to append-only log
Are you asking if this was copy/pasted from the spec?

Author: @henkbirkholz, the only difference I've added was the word **optional**.  This refers to detached content. I'll make a separate PR for updating the artifact=feed and evidence = statement body. 
Any objections to merging this change for claim=statement and ledger=append-only log?

Reviewer 3: please merge, its an improvement.

Reviewer: Again, we never store Transparent Statements inside log, if I understood correctly. We issue Transparent Statements by attaching Receipts to the Signed Statements!

### Evidence Persistence

> A SCITT instance will persist Transparent Statements to its append-only log. Any optional evidence will be persisted in associated storage. 

Reviewer 2: What would be an example of 'optional evidence'?

Author: SBOM, VEX, Scan Report, and supporting document that is either in, or referenced by the COSE Envelope

> A SCITT instance will persist Transparent Statements to its append-only log. Any optional evidence will be persisted in associated storage. 

Reviewer: ```suggestion
A SCITT instance will persist  Signed Statements to its append-only log. Any optional evidence will be persisted in associated storage. 
```
